powerauth-push-model/src/main/java/io/getlime/push/model/validator/GetApplicationDetailRequestValidator.java

```diff
@@ -34,11 +34,8 @@ public static String validate(GetApplicationDetailRequest request) {
         if (request == null) {
             return "Request must not be null.";
         }
-        if (request.getId() == null) {
-            return "ID must not be null.";
-        }
-        if (request.getId() < 1) {
```

Comment:
Just an idea. Does it make sense to validate `org.springframework.util.StringUtils.hasText(String)`?